components/Flipper/Flip/Flip.tsx

```diff
@@ -147,7 +147,7 @@ function Flip(props: Props) {
                             <b style={{ color: 'red' }}>{formatPrices(props.flip.cost)} Coins</b>
                         </p>
                     )}
-                    {settings.hideMedianPrice ? null : (
+                    {settings.hideTargetPrice ? null : (
                         <p>
                             <span>Median price: </span>
```

Comment:
this was forgotten

Reply:
I thought we agreed, that the name "hideMedianPrice" would stay?

Reply:
I still don't care about the property name, the display name is not updated